packages/io/src/types/index.ts

```diff
@@ -1,29 +1,35 @@
 import type { CKBComponents } from '@ckb-lumos/rpc/lib/types/api'
-
+import { Key } from 'path-to-regexp'
 export interface Listener<T> {
   on(listen: (obj: T) => void): void
 }
 
-export type Middleware = (ctx: Context, next: () => Promise<void>) => Promise<void>
+export interface DefaultContext {
+  /**
+   * Custom properties.
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any
+}
```

Comment:
What's the difference between `DefaultContext` and `object`

Reply:
Actually, no difference